### PR TITLE
adding db

### PR DESCRIPTION
**Pull request by Sanyyouisf**

### CODE REVIEW

**Comment by Sanyyouisf** [comment]
@drdreed
finish

File: index.html [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
-
 
+<body ng-app="MushroomApp">
+    <h2>Mushroom Mania</h2>
+    <div ng-controller="itemCtrl"> 
+        <div class="row input">
+            <div class="col-md-6 col-md-offset-3">
+                <div class="input-group">
+                    <span class="input-group-btn">
+		        		<button class="btn btn-default" type="button" ng-click="showSearch()">Go!</button>
```
Comment (DevDreed) [inline index.html:17]: Code needs formatted.

File: javascripts/main.js [hunk 2]
```diff
@@ -1 +1,46 @@
...
+            }).catch((error) => {
+                console.log("error in getItems function :", error);
+            });
+    };
+
+    getItems();
+
+    $scope.showSearch = () => {
+        $scope.showMushroomsList = true;
+        console.log(" you click showSearch");
```
Comment (DevDreed) [inline javascripts/main.js]: Remove console logs.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Just a couple minor details.

**Pull request merged**